participants.csv

```diff
@@ -1,4 +1,4 @@
 Zaccharie Ramzi,
 Guillaume Daval-Frérot,
 Mathilde Ripart,
-
+Anaïs Artiges,
```

Comment:
don't remove the whiteline

Reply:
I think it is okay now!

Reply:
no you added your name after the whiteline. The whiteline should be the last line of the file